composer.json

```diff
@@ -74,6 +76,8 @@
         "symfony/yaml": "self.version"
     },
     "require-dev": {
+        "cache/integration-tests": "dev-master",
```

Comment:
Use version `^0.6` 

A PR to this repo should not fail because we added extra tests. IMHO it is better to make a PR to update the version of the integration tests now and then.